src/main/resources/extra-data.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
It feels weird to bundle this file in /resources, but have the other locations be in configuration.yaml. Locations in this file are going to be bundled with the jar upon build. I don't think data should be bundled with the application/ jar file.